llvm/test/CodeGen/AIE/aie2/GlobalISel/inst-select-pre-post-increment.mir

```diff
@@ -23,13 +23,13 @@ body: |
     ; CHECK-NEXT: [[COPY2:%[0-9]+]]:em = COPY [[COPY1]]
     ; CHECK-NEXT: [[MOV_PD_imm10_pseudo:%[0-9]+]]:em = MOV_PD_imm10_pseudo 16
     ; CHECK-NEXT: [[MOVXM_lng_cg:%[0-9]+]]:em_as_32bit = MOVXM_lng_cg 2048
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm1:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm [[COPY]], [[COPY2]] :: (load (<16 x s16>))
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm_imm:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm_imm1:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm_imm [[VLDA_dmw_lda_w_ag_pstm_nrm1]], 0 :: (load (<16 x s16>))
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm2:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm3:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm [[VLDA_dmw_lda_w_ag_pstm_nrm_imm1]], [[MOV_PD_imm10_pseudo]] :: (load (<16 x s16>))
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm_imm2:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm_imm3:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm_imm [[VLDA_dmw_lda_w_ag_pstm_nrm3]], -2048 :: (load (<16 x s16>))
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm_imm4:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm_imm5:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm_imm [[VLDA_dmw_lda_w_ag_pstm_nrm_imm3]], 2016 :: (load (<16 x s16>))
-    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm4:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm5:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm [[VLDA_dmw_lda_w_ag_pstm_nrm_imm5]], [[MOVXM_lng_cg]] :: (load (<16 x s16>))
-    ; CHECK-NEXT: PseudoRET implicit $lr, implicit [[VLDA_dmw_lda_w_ag_pstm_nrm4]]
+    ; CHECK-NEXT: [[VLD_pstm_pseudo:%[0-9]+]]:vec256, [[VLD_pstm_pseudo1:%[0-9]+]]:ep = VLD_pstm_pseudo [[COPY]], [[COPY2]] :: (load (<16 x s16>))
+    ; CHECK-NEXT: [[VLD_pstm_imm_4x32_pseudo:%[0-9]+]]:vec256, [[VLD_pstm_imm_4x32_pseudo1:%[0-9]+]]:ep = VLD_pstm_imm_4x32_pseudo [[VLD_pstm_pseudo1]], 0 :: (load (<16 x s16>))
+    ; CHECK-NEXT: [[VLD_pstm_pseudo2:%[0-9]+]]:vec256, [[VLD_pstm_pseudo3:%[0-9]+]]:ep = VLD_pstm_pseudo [[VLD_pstm_imm_4x32_pseudo1]], [[MOV_PD_imm10_pseudo]] :: (load (<16 x s16>))
+    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm_imm:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm_imm1:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm_imm [[VLD_pstm_pseudo3]], -2048 :: (load (<16 x s16>))
+    ; CHECK-NEXT: [[VLDA_dmw_lda_w_ag_pstm_nrm_imm2:%[0-9]+]]:vec256, [[VLDA_dmw_lda_w_ag_pstm_nrm_imm3:%[0-9]+]]:ep = VLDA_dmw_lda_w_ag_pstm_nrm_imm [[VLDA_dmw_lda_w_ag_pstm_nrm_imm1]], 2016 :: (load (<16 x s16>))
+    ; CHECK-NEXT: [[VLD_pstm_pseudo4:%[0-9]+]]:vec256, [[VLD_pstm_pseudo5:%[0-9]+]]:ep = VLD_pstm_pseudo [[VLDA_dmw_lda_w_ag_pstm_nrm_imm3]], [[MOVXM_lng_cg]] :: (load (<16 x s16>))
+    ; CHECK-NEXT: PseudoRET implicit $lr, implicit [[VLD_pstm_pseudo4]]
```

Comment:
Can you also extend the test to show the limits of VLDB.postinc?

Reply:
Have added test to the show the limits of `VLD_pstm_pseudo`